gulpfile.js

```diff
@@ -104,15 +104,33 @@ gulp.task('build/zone.js', ['compile-esm'], function(cb) {
   return generateScript('./lib/browser/rollup-main.ts', 'zone.js', false, cb);
 });
 
+gulp.task('build/zone.min.js', ['compile-esm'], function(cb) {
+  return generateScript('./lib/browser/rollup-main.ts', 'zone.min.js', true, cb);
+});
+
+// Zone test bundle for the browser.
+gulp.task('build/zone-testing-bundle.js', ['compile-esm'], function(cb) {
+  return generateScript('./lib/browser/rollup-test-main.ts', 'zone-testing-bundle.js', false, cb);
+});
+
+gulp.task('build/zone-testing-bundle.min.js', ['compile-esm'], function(cb) {
+  return generateScript('./lib/browser/rollup-test-main.ts', 'zone-testing-bundle.min.js', true, cb);
+});
+
+// Zone test related files for the browser.
+gulp.task('build/zone-testing.js', ['compile-esm'], function(cb) {
+  return generateScript('./lib/testing/zone-testing.ts', 'zone-testing.js', false, cb);
+});
+
+gulp.task('build/zone-testing.min.js', ['compile-esm'], function(cb) {
```

Comment:
IIUC there is no benefit to using a minified script in tests, the only possible thing someone could do is make their debugging harder. They don't care about how many bytes are loaded in Karma. what do you think?

Reply:
yes, I agree, I will remove them.